### PR TITLE
Use config map for database server credentials

### PR DESCRIPTION
**Pull request by acbramley**
Allowing us to deploy both prod and non prod environments with the same operator.

### CODE REVIEW

**Comment by acbramley** [comment]
@nicksantamaria @nickschuch ready for a re-review

File: pkg/stub/handler.go [hunk 1]
```diff
@@ -30,64 +32,81 @@ const (
 
 func NewHandler() sdk.Handler {
```
Comment (nicksantamaria) [inline pkg/stub/handler.go]:
```
exported function NewHandler should have comment or be unexported
```

File: pkg/stub/handler.go [hunk 2]
```diff
@@ -30,64 +32,81 @@ const (
...
 	return &Handler{
-		RootUsername: os.Getenv("MYSQL_ROOT_USERNAME"),
-		RootPassword: os.Getenv("MYSQL_ROOT_PASSWORD"),
-		DbHostname:   os.Getenv("MYSQL_HOSTNAME"),
+		DefaultServer: os.Getenv("DEFAULT_SERVER"),
+		ServerNamespace: os.Getenv("WATCH_NAMESPACE"),
 	}
 }
 
 type Handler struct {
```
Comment (nicksantamaria) [inline pkg/stub/handler.go]:
```
exported type Handler should have comment or be unexported
```

File: pkg/stub/handler.go [hunk 3]
```diff
@@ -30,64 +32,81 @@ const (
...
-	RootUsername string
-	RootPassword string
-	DbHostname   string
+	DefaultServer string
+	ServerNamespace string
 }
 
 func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
```
Comment (nicksantamaria) [inline pkg/stub/handler.go]:
```
exported method Handler.Handle should have comment or be unexported
```

File: pkg/stub/handler.go (hunk 1 above)
Comment (acbramley) [reply pkg/stub/handler.go]: Fixed

File: pkg/stub/handler.go (hunk 3 above)
Comment (acbramley) [reply pkg/stub/handler.go]: Fixed

**Comment by nicksantamaria** [comment]
sealofapproval

**Pull request merged**